index.js

```diff
@@ -1,6 +1,59 @@
+const loadYaml = require('./lib/loadYaml')
 const mergeArrayByName = require('./lib/mergeArrayByName')
 
 module.exports = (robot, _, Settings = require('./lib/settings')) => {
+  async function triggerRepositoryUpdate (context, { owner, repo }) {
+    const { github } = context
+
+    const config = await loadYaml(github, {
+      owner,
+      repo,
+      path: Settings.FILE_NAME
+    })
+
+    if (config === null) {
+      robot.log.debug(`File '${Settings.FILE_NAME}' not found in '${owner}/${repo}', returning...`)
+      return
+    }
+
+    return Settings.sync(context.github, { owner, repo }, config)
+  }
+
+  robot.on([
+    'installation.created',
+    'installation.new_permissions_accepted'
+  ], async context => {
+    const { payload } = context
+    const { repositories, installation } = payload
+    const { account } = installation
+    const { login: repositoryOwner } = account
+
+    await Promise.all(repositories.map(async (repository) => {
+      const { name: repositoryName } = repository
+
+      return triggerRepositoryUpdate(context, {
+        owner: repositoryOwner,
+        repo: repositoryName
+      })
+    }))
+  })
+
+  robot.on('installation_repositories.added', async context => {
+    const { payload } = context
+    const { repositories_added: repositories, installation } = payload
```

Comment:
It seems that this is very similar to the `installation` event. Have you considered doing something like `repositories = payload.repositories | payload.repositories_added` and just combine the two events.